masu/database/aws_report_db_accessor.py

```diff
@@ -210,3 +210,13 @@ def populate_line_item_aggregate_table(self):
         self._pg2_conn.commit()
         self._vacuum_table(table_name)
         LOG.info(f'Finished updating %s.', table_name)
+
+    def mark_bill_as_finalized(self, bill_id):
+        """Mark a bill in the database as finalized."""
+        table_name = AWS_CUR_TABLE_MAP['bill']
+
+        bill = self._get_db_obj_query(table_name)\
+            .filter_by(id=bill_id)\
+            .first()
+
+        bill.finalized_datetime = self.date_accessor.today_with_timezone('UTC')
```

Comment:
Does it matter that we store two datetimes - a finalized and non-finalized one?

Would it make more sense to have a single datetime and a `finalized`  boolean that tells us the datetime and data shouldn't be modified any further?

Reply:
They're distinct events. The updates can happen at any time, and sometimes more then one time a day. That will eventually be useful to indicate the most recent time we processed data. Finalization happens only 1 time. We use it as a trigger to run an update on the entire billing period (normally we only run 3 day blocks). If we keep the finalization timestamp we can eventually surface when a billing period was finalized. Our summary tables should be considered ephemeral and the update can be run at any time, even after finalization. So we would lose that information with a single timestamp.